package.json

```diff
@@ -53,8 +53,34 @@
   "contributes": {
     "configuration": [
       {
+        "id": "snyk",
         "title": "Snyk Security",
+        "properties": {}
+      },
+      {
+        "id": "generalSettings",
+        "title": "General Settings",
+        "order": 1,
         "properties": {
+          "snyk.advanced.tokenStorage": {
+            "type": "string",
+            "enum": [
+              "Always use VS Code's secret storage"
+            ],
+            "default": "Always use VS Code's secret storage",
+            "markdownDescription": "Snyk uses VS Code's [secret storage](https://code.visualstudio.com/api/references/vscode-api#SecretStorage) to safely persist API token instead of saving it in plaintext in `settings.json`. To set the token manually, run the VS Code command [Snyk: Set Token](command:snyk.setToken)."
+          },
+          "snyk.advanced.customEndpoint": {
+            "type": "string",
+            "markdownDescription": "Sets API endpoint to use for Snyk requests. Useful for custom Snyk setups. E.g. `https://api.eu.snyk.io`.",
+            "scope": "window",
+            "pattern": "^(|(https?://)api.*.(snyk|snykgov).io)$"
+          },
+          "snyk.advanced.organization": {
+            "type": "string",
+            "markdownDescription": "Specifies an organization slug name to run tests for that organization. \n\nNote: The slug name can be extracted from the URL of your organization in the Snyk UI: `https://app.snyk.io/org/[orgslugname]`. If not specified, preferred organization as defined in your [web account settings](https://app.snyk.io/account) is used to run tests.",
```

Comment:
UUID would be better as it's unique and always works. Should we update the copy to contain instructions on how to retrieve the UUID?

Reply:
Good idea! 
OMW 💪

Reply:
My copy suggestion:

|Before|After|
|-|-|
|<img width="1320" alt="with-org-id" src="https://github.com/user-attachments/assets/c8330d13-9b00-4eec-bc91-241cf01e5a76">|<img width="1320" alt="with-slug" src="https://github.com/user-attachments/assets/3130d09a-e5ca-4e66-bdfe-db6fd5b8e3b6">|

I'm going to double-check it with Neil and Andy: